sormas-backend/src/main/java/de/symeda/sormas/backend/feature/FeatureConfigurationFacadeEjb.java

```diff
@@ -254,6 +257,35 @@ public boolean isFeatureDisabled(FeatureType featureType) {
 		return em.createQuery(cq).getSingleResult() > 0;
 	}
 
+	@Override
+	@SuppressWarnings("unchecked")
+	public boolean isPropertyValue(FeatureType featureType, FeatureTypeProperty property, boolean expectedPropertyValue) {
+
+		if (!Arrays.asList(featureType.getSupportedProperties()).contains(property)) {
+			throw new IllegalArgumentException("Feature type " + featureType + " does not support property " + property + ".");
+		}
+
+		if (!Boolean.class.isAssignableFrom(property.getReturnType())) {
+			throw new IllegalArgumentException(
+				"Feature type property " + property + " does not have specified return type " + Boolean.class.getSimpleName() + ".");
+		}
+
+		CriteriaBuilder cb = em.getCriteriaBuilder();
+		CriteriaQuery<Object> cq = cb.createQuery(Object.class);
+		Root<FeatureConfiguration> root = cq.from(FeatureConfiguration.class);
+
+		cq.where(cb.and(cb.equal(root.get(FeatureConfiguration.FEATURE_TYPE), featureType)));
+		cq.select(root.get(FeatureConfiguration.PROPERTIES));
+
+		Map<FeatureTypeProperty, Object> properties = (Map<FeatureTypeProperty, Object>) em.createQuery(cq).getSingleResult();
+
+		if (properties != null && properties.containsKey(property)) {
+			return (boolean) properties.get(property) == expectedPropertyValue;
+		} else {
+			return (boolean) property.getDefaultValue() == expectedPropertyValue;
```

Comment:
But what's the `property.getDefaultValue()` for?
Caution: `false == false` results in `true`, but is not what I would have expected!

Reply:
`property.getDefaultValue()` is supposed to return the value we want this property to have if it's not specified in the database. And `false == false` returning in `true` is exactly what is desired because the method is supposed to check whether the actual value matches the given `expectedPropertyValue`. I'll add Javadoc to the method to describe what it's supposed to do.

Reply:
I think the approach that the caller needs to provide the `expectedPropertyValue` that basically only inverts the boolean argument is superfluous and too complicated.

Reply:
I disagree. Moving the comparison logic to the backend instead of executing it in the UI in this scenario even makes the call in the UI less complicated in my opinion. If we already know which value we expect, why should we have to make the comparison in the UI instead of doing it directly in the backend, i.e. moving as much logic as possible there?

Reply:
I think having a parameter less and inverting it with `!` is the usual approach, better to read and less to type.
- `if(FeatureConfigurationFacade.isPropertyValue(FeatureType.IMMUNIZATION, FeatureTypeProperty.REDUCED)` ~ `if(FeatureConfigurationFacade.isPropertyValue(FeatureType.IMMUNIZATION, FeatureTypeProperty.REDUCED, true)`
- `if(!FeatureConfigurationFacade.isPropertyValue(FeatureType.IMMUNIZATION, FeatureTypeProperty.REDUCED)` ~ `if(FeatureConfigurationFacade.isPropertyValue(FeatureType.IMMUNIZATION, FeatureTypeProperty.REDUCED, false)`

This pattern gets even more complicated if would be about something like a String (comparison doesn't match completely, I know).

Reply:
Done, and also renamed the method to `isPropertyValueTrue` because the original name doesn't make much sense with this changed signature anymore.